casper/src/main/resources/Registry.rho

```diff
@@ -332,115 +343,518 @@ in {
+                for (realAK <<- loc) {
+                  realAK!(arg1, arg2, arg3, arg4, arg5, arg6, arg7, arg8)
+                }
+              } |
+              contract dispatcher(@arg1, @arg2, @arg3, @arg4, @arg5, @arg6, @arg7, @arg8, @arg9) = {
+                for (realAK <<- loc) {
+                  realAK!(arg1, arg2, arg3, arg4, arg5, arg6, arg7, arg8, arg9)
+                }
+              } |
+              for (@bcMap <- blessedContractLocMapCh) {
+                blessedContractLocMapCh!(bcMap.set(uri, *loc))
+              } 
+            } 
+          }|
+          for (@`rho:rchain:authKey`, @lib, @dispatcher <- insertBlessed) {
+            newInsert!(lib, `rho:rchain:authKey`, dispatcher)
+          } |
+          for (@`rho:rchain:makeMint`, @lib, @dispatcher <- insertBlessed) {
+            newInsert!(lib, `rho:rchain:makeMint`, dispatcher)
+          } |
+          for (@`rho:rchain:multiSigRevVault`, @lib, @dispatcher <- insertBlessed) {
+            newInsert!(lib, `rho:rchain:multiSigRevVault`, dispatcher)
+          } |
+          for (@`rho:lang:nonNegativeNumber`, @lib, @dispatcher <- insertBlessed) {
+            newInsert!(lib, `rho:lang:nonNegativeNumber`, dispatcher)
+          } |
+          for (@`rho:rchain:pos`, @lib, @dispatcher <- insertBlessed) {
+            newInsert!(lib, `rho:rchain:pos`, dispatcher)
+          } |
+          for (@`rho:rchain:revVault`, @lib, @dispatcher <- insertBlessed) {
+            newInsert!(lib, `rho:rchain:revVault`, dispatcher)
+          }
+        } 
+      } |
+      // --------
+      // MultiSig
+      // -------------------------------------------------------------------------------------
+      // MultiSig enables similiar functionality to a multisig vault. 
+      // pubKeys = set of public keys which have the privilege to propose and approve upgrades
+      // quorumSize = number of pubKeys member approvals needed to upgrade a contract's data
+      // -------------------------------------------------------------------------------------
+      {
+        let @quorumSize, @pubKeys <- 2, Set(
+          // The private keys and public keys are here just for testing now
+          // fc743bd08a822d544bfbe05a5663fc325039a44c8f0c7fbea95a85517da5c36b private
+          "04e2eb6b06058d10b30856043c29076e2d2d7c374d2beedded6ecb8d1df585dfa583bd7949085ac6b0761497b0cfd056eb3d0db97efb3940b14c00fff4e53c85bf".hexToBytes(),
+          // 6e88cf274735f3f7f73ec3d7f0362c439ab508427682b5bd788007aca665d810 private
+          "043405f36c6cc2327fbbdf1af442a99e9a04f650ccb415bc23d4379e63a5cb5d53f7c7861d5937db32700b001ab2024f1c078b0ba90f47af46c8c4dea33f972d7c".hexToBytes(),
+          // 87369d132ed6a7626dc4c5dfbaf41e954dd0ec55830e613a3f868c74d64a7a22 private
+          "0412eea943c13ed3e99d2de048a57dbaf4c977df83a50bf7ec79ae58255cf8d891a1cbe3cd51aeafec34241a1a9c9a54e861aa5b373667793c617c0310eab26653".hexToBytes()
```

Comment:
https://github.com/rchain/rchain/issues/3676 created an issue for configuring this